src/controllers/transfers.js

```diff
@@ -9,7 +9,43 @@ const model = require('../models/transfers')
 const InvalidUriParameterError = require('five-bells-shared/errors/invalid-uri-parameter-error')
 
 /**
- * @api {get} /transfers/:id Get local transfer object
+ * @apiGroup Test Group
+ * @apiDescription lorem ipsum dolor sit amet
```

Comment:
That was a test to see if I could create groups without associated API methods, which snuck into an in-progress commit. It got removed when it became clear that you cannot define groups that aren't associated with API methods.